sandbox/config.json

```diff
@@ -5,21 +5,21 @@
     "latency": 50,
     "logRequestHeaders": false,
     "webServices": {
-        "world": {
+        "hello/world": {
```

Comment:
Check in with Tony for what he wants these URLs to be - at the moment it looks like they'd be `hello-world/hello/world`

Reply:
Acc. to Tony:
the first level of the resource URL matches the API service name, so for the "Hello World" API, the first level of the resource URL should probably be "/hello-world" i.e. leave it as-is.